arches/app/views/api.py

```diff
@@ -524,6 +525,7 @@ def get(self, request, resourceid=None, slug=None, graphid=None):
             format = request.GET.get("format", "json-ld")
```

Comment:
we should remove the "foo" flag and add a format called "arches-json" for our current json format
the default format should be this new json format